inst/examples/app/analytics/app.R

```diff
@@ -0,0 +1,26 @@
+library(shiny)
+library(shiny.semantic)
+library(semantic.dashboard)
+library(shinyjs)
+library(tidyr)
+library(dplyr) # necessary to import %>%
+library(purrr)
+library(plotly)
+library(timevis)
+library(DT)
+# please install shiny.telemetry with all dependencies
+# install.packages("shiny.telemetry", dependencies = TRUE)
+library(shiny.telemetry)
```

Comment:
Most of these packages need to be loaded because they are in suggests instead of imports.